spec/ParseQuery.spec.js

```diff
@@ -1319,7 +1320,7 @@ describe('Parse.Query testing', () => {
       .then(done.fail)
       .catch(error => {
         equal(error.code, Parse.Error.INVALID_JSON);
-        equal(error.message, 'bad $in value');
+        expect(['bad $in value', 'bad $in value.'].indexOf(error.message) > -1).toBeTruthy();
```

Comment:
I think I mentioned this somewhere, why can't we just use `expect...contains`?

Reply:
I have reverted this and fix the test failing issue.